uint8 camera
uint8 motor
uint8 errorOpenCV
uint8 behavior
